pkg/networkmanager/interfaces.js

```diff
@@ -2567,9 +2848,16 @@ PageNetworkInterface.prototype = {
         var is_deletable = (iface && !dev) || (dev && (dev.DeviceType == 'bond' ||
                                                        dev.DeviceType == 'team' ||
                                                        dev.DeviceType == 'vlan' ||
-                                                       dev.DeviceType == 'bridge'));
+                                                       dev.DeviceType == 'bridge' ||
+                                                       dev.DeviceType == 'wifi'));
         $('#network-interface-delete').prop('hidden', !is_deletable || !managed);
 
+        function render_interface_section_separator(title) {
+            return `<tr>
```

Comment:
Does this work? I would expect that you would use things like `document.createElement("tr")`...
doc: https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement

Reply:
There are two ways to make this work. One is using jquery somewhere else (Which I was using at the time of that commit), and another one is to use innerHTML as I do use now